### DISCUSSION TITLE
Would zquickinit (an encrypted ZFS boot manager, rescue, and Proxmox installer) fit in here?

### DISCUSSION THREAD

**Discussion by midzelis** [tteck/Proxmox#1946] [General]
I just released 1.0 of https://github.com/midzelis/zquickinit. 

Its a ZFSBootMenu ([zfsbootmenu.org](https://zfsbootmenu.org)) based distro - but it embeds (optionally) Tailscale for entering your encrypted root password. It also allows you to install Proxmox directly onto a empty drive - including partitioning, creating a bootloader, encrypting a ZFS ROOT, and more. 

Would there be any interest in accepting a script that would download/install zquickinit as a Proxmox bootloader/boot-manager?

**Comment by tteck** [comment]
> Would there be any interest in accepting a script that would download/install zquickinit as a Proxmox bootloader/boot-manager?

While that project is intriguing, I have concerns about how it might complicate matters and potentially lead to significant support challenges.

**Reply by midzelis** [reply]
Thats fair. If you have time, would love it if you tried it out and let me know what you think! Thanks! 

Btw, these scripts are pretty great - so thanks!
